doc/data/messages/c/consider-using-join/good.py

```diff
@@ -0,0 +1,3 @@
+def fruits_to_string():
+    fruits = {"apple", "pear", "peach"}
+    return "".join(fruits)
```

Comment:
```suggestion
def fruits_to_string(fruits):
    return "".join(fruits)

print(fruits_to_string(["apple", "pear", "peach"]))
```

Reply:
```suggestion
print("".join(["apple", "pear", "peach"]))
```

Reply:
I think the second suggestion illustrate that the join make the function refactorable because everything is simpler.